misc/dist/ios_xcode/godot_ios.xcodeproj/project.pbxproj

```diff
@@ -197,7 +199,7 @@
 						ProvisioningStyle = Automatic;
 						SystemCapabilities = {
 							com.apple.AccessWiFi = {
-								enabled = $access_wifi;
+								enabled = 0;
```

Comment:
All games can still access wifi. This is poorly named (on Apple's part).

Reply:
yeah Apple has some strange ideas about these capabilities. Unless someone runs into a situation where this needs to be turned on, take it out.